code/__HELPERS/unsorted.dm

```diff
@@ -1634,12 +1634,19 @@ var/list/WALLITEMS = list(
 // * The dropship crash hasn't happened yet
 // * An admin hasn't disabled explosive antigrief
 // Certain areas may be exempt from this check. Look up explosive_antigrief_exempt_areas
-/proc/explosive_grief_check(var/obj/item/explosive/E)
+/proc/explosive_antigrief_check(var/obj/item/explosive/E, var/mob/user)
 	var/turf/T = get_turf(E)
```

Comment:
/E and /T are bad var names

Reply:
really man i only renamed the proc and added a pronoun......

Reply:
you touch it you fix it